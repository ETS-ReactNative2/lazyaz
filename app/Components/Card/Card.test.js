import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { Card } from './index';

test('Card component renders correctly', () => {
  const props = {
    image: '',
    title: '',
    places: 0,
    height: '10%',
  };
  const tree = renderer.create(<Card {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
