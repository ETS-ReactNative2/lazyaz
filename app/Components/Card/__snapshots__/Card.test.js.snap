// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders correctly 1`] = `
<View
  style={
    Array [
      undefined,
      Object {
        "height": 133.5,
      },
    ]
  }
>
  <Image
    source={
      Object {
        "uri": "",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": undefined,
          "width": undefined,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Array [
        undefined,
        Object {
          "height": 133.5,
        },
      ]
    }
  />
  <Text>
    
  </Text>
  <View>
    <Text>
      You can use this service for delivery: documents, parcels, courier service and confidential paper delivery.
    </Text>
    <View>
      <BVLinearGradient
        colors={
          Array [
            undefined,
            undefined,
          ]
        }
        endPoint={
          Object {
            "x": 1,
            "y": 1,
          }
        }
        locations={null}
        startPoint={
          Object {
            "x": 0,
            "y": 1,
          }
        }
      >
        <View
          accessible={true}
          isTVSelectable={true}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Array [
                undefined,
                Object {
                  "flexDirection": null,
                  "height": 48,
                  "width": 375,
                },
              ]
            }
          >
            <Text
              style={
                Array [
                  Array [
                    undefined,
                    Object {
                      "lineHeight": 48,
                    },
                  ],
                  Object {
                    "color": undefined,
                  },
                ]
              }
            >
              MAKE ORDER
            </Text>
          </View>
        </View>
      </BVLinearGradient>
    </View>
  </View>
</View>
`;
